### ISSUE TITLE
Implements tests for the new components

### ISSUE THREAD

**Issue by ddelpiano** [VirtualFlyBrain/geppetto-vfb#565]
As first step I will put specs here for the Loader only since the graph will go through some changes in this sprint, so better to put those on hold.

For the loader it would be nice to have the few scenarios below:

- Open the url with suffix **geppetto** (e.g. https://localhost:8080/org.geppetto.frontend/geppetto), 
- Check that the loader will pop up with the numbers 1/1 (means we are loading a single instance)
- Check that as soon as all the components are loaded (3d Viewer, Slice Viewer, Term Info are in focus by default, so we should check them) and the instance data are displayed (on term info check inside the nested div we can see some info related to the id VFB_00017894, for the 3d viewer check that Instance[VFB_00017894].isVisible return true, for the Slice viewer need to check what is the method to check if data are loaded) the loader is then removed.

- Open the url with suffix **geppetto?id=VFB_00017894&i=VFB_00000001** (e.g. https://localhost:8080/org.geppetto.frontend/geppetto?id=VFB_00017894&i=VFB_00000001), 
- Check that the loader will pop up with the numbers 2 as total of the instances that we are loading.
- Check that as soon as all the components are loaded (3d Viewer, Slice Viewer, Term Info are in focus by default, so we should check them) and the instance data are displayed (on term info check inside the nested div we can see some info related to the id VFB_00017894 since using id= for this id, for the 3d viewer check that Instance[*].isVisible return true (where * is replaced by each ID we are loading), for the Slice viewer need to check what is the method to check if data are loaded) the loader is then removed.

- Open the url with suffix **geppetto?id=FBbt_00003678&i=VFB_00017894,VFB_00030624,FBbt_00003678** (e.g. https://localhost:8080/org.geppetto.frontend/geppetto?id=FBbt_00003678&i=VFB_00017894,VFB_00030624,FBbt_00003678), 
- Check that the loader will pop up with the numbers 3 as total of the instances that we are loading.
- Check that as soon as all the components are loaded (3d Viewer, Slice Viewer, Term Info are in focus by default, so we should check them) and the instance data are displayed (on term info check inside the nested div we can see some info related to the id FBbt_00003678 since using id= for this id, for the 3d viewer check that Instance[*].isVisible return true (where * is replaced by each ID we are loading, we need to check the visual capabilities for the all ids in this case since FBbt_00003678 does not have any), for the Slice viewer need to check what is the method to check if data are loaded) the loader is then removed.

This second part needs to happen after #567.

Tests for graph and tree components:

- Open the url with suffix **geppetto** (e.g. https://localhost:8080/org.geppetto.frontend/geppetto), 
- Select the Tree/Graph component
- Check the root element displayed by the Tree/Graph component and that this match with the ID displayed by the focus term (VFB_00017894).
- Add another ID to the scene (e.g. FBbt_00003678)
- Check that the root displayed by the Graph is still the previous one.
- Check that the root displayed by the Tree component changed with FBbt_00003678.
- Click the sync button in the Graph component
- Check that the root displayed by the Graph component changed with FBbt_00003678.

Forr the graph component we should test also that the the zoom in/zoom out/reset position buttons are performing as expected, I am not sure what we can do here with jest, @jrmartin do you know if we can test this inside the canvas or do you think we will need some sort of data to be populated in the global/window context and then check them through javascript console?
This part should be implemented also for the circuit browser.

Layer component:
- We need to add one anatomy instance and one neuron instance
- check that the 2 instances added appear in the layer component
- use every action that we have in the menu for both the instances and check that the action has been executed.

Circuit browser:
To be discussed, we can do this after the last pending cards related to him will be implemented.

FocusTerm:
- Load standard template
- load medulla class (through stack viewer or search component, id FBbt_00003748)
- check that the label in the focus term changed from the previous
- click on the focus term
- check that one of the menu item opens the query builder
- select back the template ID
- check that the label in the focus term changed from the previous

**Comment by ddelpiano** [comment]
@jrmartin we need to park this card until the next improvements spec for the graph will be ready.
Removing you from the card temporarily just to avoid confusion.
